assets/js/translator.js

```diff
@@ -350,13 +339,391 @@ if(modeEnabled('translation')) {
             persistChoices('translator');
         });
 
+        initChainGraph();
```

Comment:
`setupChainGraph` just to be consistent and since it's only called here, could you also just add `refreshChosenPath()` to the end of it?